crates/workspace/src/notifications.rs

```diff
@@ -213,6 +213,7 @@ impl Render for LanguageServerPrompt {
             .id("language_server_prompt_notification")
             .elevation_3(cx)
             .items_start()
+            .justify_between()
```

Comment:
added this because the close icon was flushing to the top right corner

Reply:
Did it fix anything? If not, let's remove it?

Reply:
before adding this, the cross icon would be right beside the title. So i added justify between so that the cross is always at the top right corner.